internal/olm/operator/internal/registry_pod.go

```diff
@@ -0,0 +1,313 @@
+// Copyright 2020 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package olm
+
+import (
+	"bytes"
+	"context"
+	"errors"
+	"fmt"
+	"path/filepath"
+	"strings"
+	"text/template"
+	"time"
+
+	"github.com/operator-framework/operator-sdk/internal/flags"
+	"github.com/spf13/viper"
+	corev1 "k8s.io/api/core/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/cluster-api/util/container"
+)
+
+// BundleAddModeType - type of BundleAddMode in RegistryPod struct
+type BundleAddModeType = string
+
+const (
+	defaultIndexImage        = "quay.io/operator-framework/upstream-opm-builder:latest"
+	defaultContainerName     = "registry-grpc"
+	defaultContainerPortName = "grpc"
+	defaultGRPCPort          = 50051
+
+	// SemverBundleAddMode - bundle add mode for semver
+	SemverBundleAddMode BundleAddModeType = "semver"
+
+	// ReplacesBundleAddMode - bundle add mode for replaces
+	ReplacesBundleAddMode BundleAddModeType = "replaces"
+)
```

Comment:
Move these types to their own section
```suggestion
// BundleAddModeType - type of BundleAddMode in RegistryPod struct
type BundleAddModeType = string

const (
	// SemverBundleAddMode - bundle add mode for semver
	SemverBundleAddMode BundleAddModeType = "semver"
	// ReplacesBundleAddMode - bundle add mode for replaces
	ReplacesBundleAddMode BundleAddModeType = "replaces"
)

const (
	defaultIndexImage        = "quay.io/operator-framework/upstream-opm-builder:latest"
	defaultContainerName     = "registry-grpc"
	defaultContainerPortName = "grpc"
	defaultGRPCPort          = 50051
)
```